.travis.yml

```diff
@@ -1,11 +1,8 @@
 language: scala
 sudo: false
 scala:
-  - 2.10.6
```

Comment:
Build for 2.12.0 fails because at least finagle artifacts are not yet published for it.